test/interp/simd-binary.txt

```diff
@@ -80,6 +80,36 @@
     v128.const i32 0x00ff0001 0x00040002 0x00000003 0x00000004
     v128.const i32 0x00020001 0x00fe0002 0x00000003 0x00000004
     i64x2.sub)
+
+  ;; i8x16 mul
+  (func (export "i8x16_mul_0") (result v128)
+    v128.const i32 0x00000001 0x00000002 0x00000003 0x00000004
+    v128.const i32 0x00000001 0x00000002 0x00000003 0x00000004
+    i8x16.mul)
+  (func (export "i8x16_mul_1") (result v128)
+    v128.const i32 0x00ff0001 0x00040002 0x00000003 0x00000004
+    v128.const i32 0x00020001 0x00fe0002 0x00000003 0x00000004
+    i8x16.sub)
```

Comment:
should be `i8x16.mul`

Reply:
@binji 
Thanks! Fixed it, PTAL!

@arunetm FYI!